unicodetools/src/main/java/org/unicode/props/PropertyParsingInfo.java

```diff
@@ -851,6 +851,13 @@ private static void parsePropertyValueFile(
                                                 <= 0
                                 ? new PropertyUtilities.Joiner("|")
                                 : null;
+                final var originalMultivaluedSplit = propInfo.multivaluedSplit;
+                // The first version of kPrimaryNumeric had spaces in values.
+                if (propName.equals("kPrimaryNumeric")
+                        && indexUnicodeProperties.ucdVersion.compareTo(VersionInfo.UNICODE_4_0)
```

Comment:
nit: second check for ucdVersion <= 4.0 in the same block...

Reply:
Yes, but it doesn’t factor nicely since I am declaring things.

(Also, morally that second one is == 3.2, since Unicode 4.0 did not update Unihan, whereas the other one really covers multiple versions.)